src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/SQLFunctionsIT.java

```diff
@@ -312,6 +317,86 @@ public void castIntFieldToDoubleWithAliasJdbcFormatGroupByTest() {
         }
     }
 
+    @Test
+    public void castFieldToDatetimeWithoutAliasTest() throws IOException {
+        String query = "SELECT CAST(insert_time AS DATETIME) FROM " + TestsConstants.TEST_INDEX_ONLINE +
```

Comment:
I see that index online doesn't have predefined mapping. Could you check if field `insert_time` is already a date in ES? If so, I think we need to test against fields of different type, ex. keyword. I assume that's the point of `CAST` statement.

Reply:
You're right- will find other fields to test the Datetime cast